pkgs/tools/misc/yubico-piv-tool/default.nix

```diff
@@ -25,6 +25,9 @@ stdenv.mkDerivation rec {
 
   configureFlags = [ "--with-backend=${if withApplePCSC then "macscard" else "pcsc"}" ];
 
+  # Workaround with failing tests compilation because of unitialized variables
+  CFLAGS = [ "-Wno-error=maybe-uninitialized" ];
```

Comment:
I just double checked this and found maybe a better solution:

In CMakeLists.txt lin 100 we have ``set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Werror")``. We should remove that entire line to prevent future regressions like this when for example updating the c compiler.

Reply:
Or just remove the `-Werror` part? The other ones may be still useful to see if we run into issues in the future.

Reply:
I remove the `CFLAGS` line and added

```nix
postPatch = ''
  substituteInPlace CMakeLists.txt --replace "-Werror" ""
'';
```

to pkgs/tools/misc/yubico-piv-tool/default.nix

What do you think?